source/extensions/filters/listener/tls_inspector/config.cc

```diff
@@ -0,0 +1,46 @@
+#include <string>
+
+#include "envoy/registry/registry.h"
+#include "envoy/server/filter_config.h"
+
+#include "extensions/filters/listener/tls_inspector/tls_inspector.h"
+#include "extensions/filters/listener/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+namespace TlsInspector {
+
+/**
+ * Config registration for the TLS inspector filter. @see NamedNetworkFilterConfigFactory.
+ */
+class TlsInspectorConfigFactory : public Server::Configuration::NamedListenerFilterConfigFactory {
+public:
+  // NamedListenerFilterConfigFactory
+  Server::Configuration::ListenerFilterFactoryCb
+  createFilterFactoryFromProto(const Protobuf::Message&,
+                               Server::Configuration::ListenerFactoryContext& context) override {
+    ConfigSharedPtr config(new Config(context.scope(), Config::TLS_MAX_CLIENT_HELLO));
```

Comment:
Unless I'm reading C++ wrong, 2nd parameter is now optional and defaults to Config::TLS_MAX_CLIENT_HELLO, so this isn't needed here.